src/main/java/com/epam/lab/collections2/consolemenu/Main.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.lab.collections2.consolemenu;
+
+public class Main {
+    public static void main(String[] args) {
+        ConsoleMenu consoleMenu = new ConsoleMenu();
+        consoleMenu.myView();
```

Comment:
Note method name should be a verb because method do something class name should be a noun  
so myView() - getMyView, printMyView, initMyView  demo -> runDemo, startDemo ...